tests/post/features/monitoring.feature

```diff
@@ -6,12 +6,12 @@ Feature: Monitoring is up and running
 
     Scenario: Expected Pods
         Given the Kubernetes API is available
-        Then we have 1 running pod labeled 'k8s-app=prometheus-operator' in namespace 'monitoring' on node 'bootstrap'
-        And we have 2 running pod labeled 'prometheus=k8s' in namespace 'monitoring' on node 'bootstrap'
-        And we have 3 running pod labeled 'alertmanager=main' in namespace 'monitoring' on node 'bootstrap'
-        And we have 1 running pod labeled 'app=grafana' in namespace 'monitoring' on node 'bootstrap'
-        And we have 1 running pod labeled 'app=kube-state-metrics' in namespace 'monitoring' on node 'bootstrap'
-        And we have 1 running pod labeled 'app=node-exporter' in namespace 'monitoring' on node 'bootstrap'
+        Then we have 1 running pod labeled 'k8s-app=prometheus-operator' in namespace 'monitoring'
```

Comment:
You should explain why we are removing this in the commit message.

Reply:
done

Reply:
Where? I still see no info in the commit message (only the title ` Remove node checking from monitoring tests `).

Reply:
I don't know what I've done, but it seems that I didn't push this change :p
Now it's done... for real. :)